src/client/Microsoft.Identity.Client/ApiConfig/WebView2Options.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.Core;
+using Microsoft.Identity.Client.Internal.Logger;
+using Microsoft.Identity.Client.PlatformsCommon.Factories;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// Options for using the modern Windows embedded browser WebView2. 
+    /// For more details see https://aka.ms/msal-net-webview2
+    /// </summary>
+#if !SUPPORTS_WEBVIEW2
+    [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
+#endif
+    public class WebView2Options
+    {
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        public WebView2Options()
+        {
+            ValidatePlatformAvailability();
+        }
+
+        /// <summary>
+        /// Forces a static title to be set on the window hosting the browser. If not configured, the widow's title is set to the web page title.
+        /// </summary>
+        public string Title { get; set; }
+
+        /// <summary>
+        /// Allows use of a static WebView2 runtime. If unset, the WebView2 API tries to locate the globally available version.
```

Comment:
I don't understand why and what this is?
Do we want to write:

It is possible for applications to bundle a fixed version of the runtime, and ship it side-by-side.
For this you need to tell MSAL (so it can tell WebView2) where to find the runtime bits by setting this property. If you don't set it, MSAL will attempt to use a system-wide "evergreen" installation of the runtime."